src/common/harvest/harvest.js

```diff
@@ -177,7 +176,7 @@ export class Harvest extends SharedContext {
       encodeParam('v', VERSION),
       transactionNameParam(info),
       encodeParam('ct', runtime.customTransaction),
-      '&rst=' + now(),
+      '&rst=' + this.timeKeeper.now(),
```

Comment:
This would be the place that `sharedContext` would come into play with the comment above, ie. `this.sharedContext.timeKeeper`